diff --git a/tools/iEDA/script/iRT_script/run_iRT.tcl b/tools/iEDA/script/iRT_script/run_iRT.tcl
index 8b59ab1..22afe56 100755
--- a/tools/iEDA/script/iRT_script/run_iRT.tcl
+++ b/tools/iEDA/script/iRT_script/run_iRT.tcl
@@ -65,7 +65,7 @@ def_init -path $INPUT_DEF
 init_rt -temp_directory_path $TOOL_REPORT_DIR \
         -bottom_routing_layer "met1" \
         -top_routing_layer "met4" \
-        -thread_number $NUM_THREADS
+        -thread_number $NUM_THREADS -enable_fast_mode 1

 run_rt
